tests/rkt_paths_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build host coreos src
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+// TestPaths checks whether access to paths like /proc/sysrq-trigger are
+// restricted
+func TestPaths(t *testing.T) {
```

Comment:
This is partly overlapping with https://github.com/lucab/rkt/blob/to-upstream/seccomp-guide/tests/rkt_mount_test.go#L109 and I think the two can be merged (in whichever direction you prefer).